packages/expo-cli/src/credentials/views/SetupAndroidKeystore.ts

```diff
@@ -0,0 +1,18 @@
+import { AndroidCredentials } from '@expo/xdl';
+
+import { DownloadKeystore, UpdateKeystore } from './AndroidKeystore';
+
+import { Context, IView } from '../context';
+
+export class SetupAndroidKeystore implements IView {
+  constructor(private experienceName: string) {}
+
+  async open(ctx: Context): Promise<IView | null> {
+    const { keystore, keystorePassword, keyAlias, keyPassword } =
+      (await ctx.android.fetchKeystore(this.experienceName)) ?? {};
+    if (!keystore || !keystorePassword || !keyAlias || !keyPassword) {
+      return new UpdateKeystore(this.experienceName);
+    }
+    return null;
```

Comment:
```suggestion
    const keystore = await ctx.android.fetchKeystore(this.experienceName);
    if (!keystore) {
      return new UpdateKeystore(this.experienceName);
    } else {
      return null;
    }
```

Reply:
Destructuring the result of`ctx.android.fetchKeystore` doesn't make sense. 

The result is either null or Keystore:
`public async fetchKeystore(experienceName: string): Promise<Keystore | null> {`

and this is the Keystore type definition:
```
type Keystore = {
    keystore: string;
    keystorePassword: string;
    keyPassword: string;
    keyAlias: string;
};
```

Reply:
In case of invalid data in db